runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/state/FlinkStateInternals.java

```diff
@@ -571,6 +573,27 @@ public void clear() {
         throw new RuntimeException("Error clearing state.", e);
       }
     }
+
+    @Override
```

Comment:
This change is nice to have. It should be in a separate commit. Can you please adjust the commit history so that each independent change is in a clearly labeled commit, and I will not squash them. In other words, merge the spotless chore and other non-meaningful commits into the other commits so that the commits in the PR are each meaningful on their own.

Reply:
Got it!

I'll clean up the commit history by merging formatting changes into meaningful commits and keeping independent changes separate.